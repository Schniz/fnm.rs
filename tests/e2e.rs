#[macro_use]
mod shellcode;

mod feature_tests;
