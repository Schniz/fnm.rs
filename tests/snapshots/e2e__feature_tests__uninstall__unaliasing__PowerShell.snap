---
source: tests/e2e.rs
expression: "&source.trim()"
---
$ErrorActionPreference = "Stop"
fnm env | Out-String | Invoke-Expression
fnm install v10.11
fnm alias v10.11 test-alias
$($__out__ = $(fnm ls | Select-String 'test-alias'); echo $__out__; if ($__out__ -eq $null){ exit 1 } else { $__out__ })
$($__out__ = $(fnm uninstall test-alias | Select-String 'Unaliased'); echo $__out__; if ($__out__ -eq $null){ exit 1 } else { $__out__ })
$($__out__ = $(fnm ls | Select-String -NotMatch 'test-alias'); echo $__out__; if ($__out__ -eq $null){ exit 1 } else { $__out__ })
